packages/playground/src/components/traces/TracesList/TraceElement.tsx

```diff
@@ -43,43 +50,67 @@ export function TraceElement({ trace }: { trace: Trace }) {
     return null;
   }
 
-  const incomingRequest = waterfall[0];
+  // HACK - Need a better typesafe way to get the incoming request span
+  const incomingRequest = waterfall[0] as SpanWithVendorInfo;
```

Comment:
There are no checks to see if there actually are any items in the array, is there?